apistar/settings.py

```diff
@@ -21,11 +18,14 @@ def get(self, indexes: Union[str, List[str]], default: Any=None) -> Any:
         return value
 
 
-class Setting(object):
-    def __new__(cls, *args):
-        assert len(args) == 1
-        return args[0]
+@builder
+def build_settings(app: app.App) -> Settings:
+    return Settings(app.settings)
 
-    @classmethod
-    def build(cls, arg_name: ArgName, settings: Settings):
-        return settings.get(arg_name)
+
+Setting = object
```

Comment:
Nope, because later on this gets passed through `issubclass`, which expects a Class, not a type.